austation/code/modules/power/rbmk.dm

```diff
@@ -564,6 +577,10 @@ The reactor CHEWS through moderator. It does not do this slowly. Be very careful
 	if(action == "input")
 		var/input = text2num(params["target"])
 		reactor.desired_k = CLAMP(input, 0, 3)
+		if(reactor.log_delay < world.time) //Used to avoid spam
+			reactor.log_delay = world.time + 2 SECONDS
```

Comment:
Because the target criticality can be used as a slider, this will only show the first decimal increase in most cases.

Reply:
For example, “ADMIN: Reactor rods set to 1.1” when it was actually set to 3